db/data_migration/20150424094040_republish_coming_soon.rb

```diff
@@ -0,0 +1 @@
+DataHygiene::PublishingApiRepublisher.new(Edition.scheduled).perform
```

Comment:
I'm not sure this is right. Only scheduled editions without an existing published edition will have a coming_soon item.